backend/isoko-backend/src/handlers/business/put-edit-business-page.js

```diff
@@ -9,11 +60,76 @@ exports.putEditBusinessPageHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const { businessId } = event.pathParameters; 
+
+   if (businessId == null) {
+      console.log("error"); 
+      throw new Error(
+         `Missing query parameter 'businessId'. Request URL format: PUT/business/{businessId}`
+      );
+   }
+
+   const requestBody = event.body && JSON.parse(event.body);
+   const fieldNames = _.keys(requestBody); 
+   console.log(fieldNames); 
+   // following fields should not be modified 
+   const restrictedFields = ['pk', 'sk', 'businessId', 'reviews', 'rating', 'numReviews', 'claimed']; 
+   
+
+   let exprAttrVals = {}; 
+   const updateExpr = buildUpdateExpression(fieldNames, requestBody, exprAttrVals); 
+   console.log(updateExpr); 
+   console.log(exprAttrVals); 
```

Comment:
nit: theres a bunch of console logs here from what looks like testing. I would say remove them but also if we think it is helpful to keep these logs can we switch them to console.info's (for consistency w other endpoints) and maybe add some more description on what is being logged